frontend/tests/utils/wmgUtils.ts

```diff
@@ -213,30 +271,25 @@ export const checkSourceData = async (page: Page) => {
 
   return n;
 };
+
 export const checkPlotSize = async (page: Page) => {
-  //get the number of rows on the data plot
-  const n = await page.locator('[data-zr-dom-id*="zr"]').count();
-  let sumOfHeights = 0;
-  for (let i = 0; i < n; i++) {
```

Comment:
no need to sum all heights, since querying the DOM is slow. We just need to verify the first element has height > 0, then just query how many such elements exist